src/main/java/org/jabref/logic/journals/PredatoryJournalLoader.java

```diff
@@ -0,0 +1,140 @@
+package org.jabref.logic.journals;
+
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+import org.jabref.logic.net.URLDownload;
+
+import kong.unirest.UnirestException;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class PredatoryJournalLoader {
+    private static class PJSource {
+        URL URL = null;
+        final String ELEMENT_REGEX;
```

Comment:
Variables, fields, ... should be lower case. Please change to lower case. (also at the constructor below)